chainindexer.go

```diff
@@ -239,7 +240,7 @@ func (a *addrIndexer) indexManager() {
 			addrIndex, err := a.indexBlockAddrs(indexJob.blk)
 			if err != nil {
 				adxrLog.Errorf("Unable to index transactions of"+
-					" block %v", err)
+					" block, %v", err)
```

Comment:
I'd prefer if this was a colon for consistency with the rest of the logged errors